src/commands/general/whois.ts

```diff
@@ -0,0 +1,24 @@
+import Command, { SendFunction } from '../../util/BaseCommand';
+import SentinelClient from '../../client/SentinelClient';
+import { Message } from 'discord.js';
+import CommandArguments from '../../util/CommandArguments';
+import Util from '../../util';
+
+export default class guildStatsCommand extends Command {
```

Comment:
```suggestion
export default class WhoIsCommand extends Command {
```
same as previous, better class name too